docs/conf.py

```diff
@@ -269,6 +269,10 @@
 # -- Sphinx Gallery ------------------------------------------------------------
 # JSOC email os env
 # see https://github.com/sunpy/sunpy/wiki/Home:-JSOC
+# use "export JSOC_EMAIL='your_email@example.com'"
```

Comment:
This file does not need to be changed.

Reply:
Thanks for the information @nabobalis. Let me fix it.